ground/src/main/java/com/google/android/ground/persistence/local/room/dao/SurveyDao.kt

```diff
@@ -29,8 +28,5 @@ interface SurveyDao : BaseDao<SurveyEntity> {
 
   @Transaction
   @Query("SELECT * FROM survey WHERE id = :id")
-  fun getSurveyById(id: String): Maybe<SurveyEntityAndRelations>
-
-  @Query("SELECT * FROM survey WHERE id = :id")
-  suspend fun getSurveyByIdSuspend(id: String): SurveyEntityAndRelations?
+  suspend fun getSurveyById(id: String): SurveyEntityAndRelations?
```

Comment:
Should we make the naming convention consistent? Here we use `get*` and in other places we use `find*`. I'm fine with either, but we should stick to one scheme.

Reply:
Done